packages/client/lib/types.ts

```diff
@@ -10,3 +12,36 @@ export interface Bootnode {
   port: number
 }
 export type BootnodeLike = string | string[] | Bootnode | Bootnode[]
+
+// QHeap types. @types/qheap does not exist, so we create a custom interface here.
```

Comment:
nice. another option here is to make a `types/qheap.d.ts` file and then include it in the top of the file you import qheap with `/// <reference path="../types/qheap.d.ts" />`

Reply:
in some cases i find it also needs to fed into the tsconfig [`typeRoots`](https://www.typescriptlang.org/tsconfig#typeRoots) if done this way ^

Reply:
Thanks, I did not know that this was a feature. I will keep it in `types.ts` for now, though.